app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AnimalsController < ApplicationController
+  def index
+
+  end
+  
+  def show
+
+  end
+
+  def new
```

Comment:
if we have a new action we also need a create, the new is the form that we will get the animals data, and the create is the action that will build the animal